callback_plugins/default_plus_summary.py

```diff
@@ -0,0 +1,94 @@
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Ansible callback plugin.
+'''
+
+from ansible.plugins.callback.default import CallbackModule as CallbackModule_default
+from ansible import constants as C
+from ansible.utils.color import stringc
+
+
+class CallbackModule(CallbackModule_default):
+    '''
+    This is like the default callback plugin, but also stores results and
+    summarizes failures.
+    '''
+
+    CALLBACK_VERSION = 2.0
+    CALLBACK_TYPE = 'stdout'
```

Comment:
@brenton Yes, the default plugin would still run, aggregate plugins are meant to provide supplemental output.

Thinking from a practical use case, it seems like being able to collect and aggregate output would be useful independent of a separate 'tool' that would just provide limited output. Maybe having two callback plugins, one that would provide the aggregation and summarization and one that could replace the default callback plugin would be the way to go to achieve both use cases?